src/blocks/notifications/Notifications.tsx

```diff
@@ -0,0 +1,149 @@
+import { FC, useEffect } from 'react';
```

Comment:
<img width="1508" alt="Screenshot 2024-08-09 at 2 51 26 PM" src="https://github.com/user-attachments/assets/e747c6cc-dd32-4398-9830-6303cc65612d">
<img width="456" alt="Screenshot 2024-08-09 at 2 53 14 PM" src="https://github.com/user-attachments/assets/83c3b916-1794-4233-86ab-e9b03e8a58e9">


- The position of the notification component seems incorrect. Too much spacing on the right side.
- The notification component should have a fixed height and width take make it look consistent. Right now the width and height changes according to text length.